specs/linux/selinux_settings.table

```diff
@@ -0,0 +1,11 @@
+table_name("selinux_settings")
+description("Track active SELinux settings.")
+schema([
+    Column("scope", TEXT, "Scope", index=True),
+    Column("key", TEXT, "Key", index=True),
```

Comment:
Key is the name of an SELinux setting, where do these come from, are they well-known or will they be different per system, per kernel version, etc. Try to describe what to expect.